### ISSUE TITLE
Incompatible with RainLab.Translate v2

### ISSUE THREAD

**Issue by SamBrishes** [jan-vince/smallcontactform#149]
Hello,

using the latest OctoberCMS v3.1 release together with RainLab.Translate v2 the following error occurs while scanning for messages via the RainLab.Translate admin menu:

```
"Class "RainLab\Translate\Models\Locale" not found" on line 327 of 
[root]/plugins/janvince/smallcontactform/models/Settings.php
```

~ Sam.

**Comment by jan-vince** [comment]
Hi Sam and thank you - I will look at this as soon as I have some spare time!

**Comment by LZaplata** [comment]
Hi, I have created pull request with compatibility fix.

https://github.com/jan-vince/smallcontactform/pull/152

**Comment by jan-vince** [comment]
Merged and will be in 1.60.0 version. Thank you!
